QUICKSTART-VMWARE.md

```diff
@@ -0,0 +1,258 @@
+
+You'll first need an AWS account, and AWS CLI installed.
+Then you'll create a service role in that account to [grant AWS STS trust to the AWS Systems Manager service](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-service-role.html).
+```
+cat <<EOF > ssmservice-trust.json
+{
+    "Version": "2012-10-17",
+    "Statement": {
+        "Effect": "Allow",
+        "Principal": {
+            "Service": "ssm.amazonaws.com"
+        },
+        "Action": "sts:AssumeRole"
+    }
+}
+EOF
+
+# Create the role using the above policy
+aws iam create-role \
+    --role-name SSMServiceRole \
+    --assume-role-policy-document file://ssmservice-trust.json
+
+# Attach the policy enabling the role to create session tokens
+aws iam attach-role-policy \
+    --role-name SSMServiceRole \
+    --policy-arn arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore  
+```
+
+Once the above is created, we can use the role to create an activation code and ID.
+```
+# `default-instance-name` and `description` are optional
+export SSM_ACTIVATION="$(aws ssm create-activation \
+  --default-instance-name MyVMwareVMs \
+  --description "Activation for VMware VMs" \
+  --iam-role SSMServiceRole \
+  --registration-limit 100 \
+  --region us-west-2 \
+  --output json)"
+```
+
+Using the above activation data we can create our user data to pass to the control container
+```
+SSM_ACTIVATION_ID="$(jq -r '.ActivationId' <<< ${SSM_ACTIVATION})"
+SSM_ACTIVATION_CODE="$(jq -r '.ActivationCode' <<< ${SSM_ACTIVATION})"
+CONTROL_USER_DATA="$(echo '{"ssm":{"activation-id":"'${SSM_ACTIVATION_ID}'","activation-code":"'${SSM_ACTIVATION_CODE}'","region":"us-west-2"}}' | base64 -w0)"
+
+cat <<EOF >>userdata.toml
+[settings.host-containers.control]
+enabled = true
+user-data = "${CONTROL_USER_DATA:-<CONTROL_USER_DATA>}"
+EOF
+```
+
+### Setting user data via "guestinfo" interface
+**Note: You must set these values before you start the VM for the first time!**
+
+VMware allows you to set some extended attributes of your VM, which your VM can then access via a "guestinfo" interface.
+These extended attributes are `guestinfo.userdata` and `guestinfo.userdata.encoding`.
+
+`guestinfo.userdata` may be passed as base64, gzipped base64, or (least desirable) raw TOML.
+Valid values for `guestinfo.userdata.encoding` are: `base64`, `b64`, `gzip+base64`, and `gz+b64`.
+
+Given the above file `userdata.toml`, base64 encode and set user data for your VM:
+```
+export BR_USERDATA=$(base64 -w0 userdata.toml)
+govc vm.change -vm <YOUR VM NAME> -e guestinfo.userdata="${BR_USERDATA}" -e guestinfo.userdata.encoding="base64"
+```
+
+You can check that your user data was set:
+```
+govc vm.info -e -r -t <YOUR VM NAME>
+```
+
+## Launch!
+Once you've created your user data, given your VM a way to access it via guestinfo or CD-ROM, you can launch a Bottlerocket VM in your cluster!
```

Comment:
I'm not sure about this last sentence. Since we don't mention how to pass user data via CD-ROM in the quickstart maybe this could simply be:
> "Once you've applied your user data via guestinfo, you can launch a Bottlerocket VM in your cluster!"